containers/elabctl/README.md

```
@@ -0,0 +1,29 @@
+# elabctl
+
+This script is used to manage an eLabFTW instance.
+
+#### See the main [documentation](https://doc.elabftw.net).
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h4

(MD001, heading-increment)